### DISCUSSION TITLE
Welcome to vivisect Discussions!

### DISCUSSION THREAD

**Discussion by atlas0fd00m** [atlas0fd00m/vivisect#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bat-serjo** [comment]
Can Solver do concolic analysis?

**Comment by bat-serjo** [comment]
root@rocket:/home/serj/_o/netstock/vivisect-d00m/vivisect# python3 vdbbin.py -Q
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
[7201:7201:1014/192458.629079:ERROR:zygote_host_impl_linux.cc(89)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
root@rocket:/home/serj/_o/netstock/vivisect-d00m/vivisect# python3 vdbbin.py -Q --no-sandbox
usage: vdbbin [options] [platformopt=foo, ...]
vdbbin: error: unrecognized arguments: --no-sandbox
